alpha-lab/phone-neon/index.md

```diff
@@ -1,118 +1,103 @@
 ---
 title: Uncover gaze behaviour on phone screens with Neon
 description: "Evaluate Neon's accuracy on phone screens"
-permalink: /alpha-lab/phone-screens
+permalink: /alpha-lab/phone-neon
+meta:
+  - name: twitter:card
+    content: player
+  - name: twitter:image
+    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
+  - name: twitter:player
+    content: "https://www.youtube.com/embed/gp5O1uskDME"
+  - name: twitter:width
+    content: "1280"
+  - name: twitter:height
+    content: "720"
+  - property: og:image
+    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
 tags: [Neon, Cloud]
 ---
 
 <script setup>
 import TagLinks from '@components/TagLinks.vue'
 </script>
 
-# Uncover gaze behaviour on phone screens with Neon
+# Uncover Gaze Behaviour on Phone Screens With Neon
 
 <TagLinks :tags="$frontmatter.tags" />
 
 <Youtube src="gp5O1uskDME"/>
 
-Have you ever wondered what your eyes focus on when scrolling through your favourite app?
+What catches your attention and how do you visually navigate through the interface/page of a mobile app? This article shows you the tools you can use to address these questions. 
```

Comment:
```suggestion
What catches your attention and how do you visually navigate through the interface of a mobile app? This article we use Neon, our wearable eye tracker, and all the other tools you need to address these questions. 
```

Reply:
Thanks. I've made a minor correction in the second sentence: "What catches your attention and how do you visually navigate through the interface of a mobile app? In this article, we use Neon, our wearable eye tracker, and all the other tools you need to address these questions."

Reply:
sure, sounds good.